rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -445,6 +454,8 @@ private int retryGetEndStatus(URI endPath, boolean compensate) {
 
                 // 200 and 410 are the only valid response code for reporting the participant status
                 if (response.getStatus() == Response.Status.GONE.getStatusCode()) {
+                    // TODO shouldn't 410 be handled in a similar manner as 200 on retries?
+                    status = status.equals(ParticipantStatus.Compensating) ? ParticipantStatus.Compensated : ParticipantStatus.Completed;
```

Comment:
Does it block the merging? Where will resolution of the TODO come from?

Reply:
If @mmusgrov agrees then I will move it into a separate JBTM.

Reply:
Thanks